core-services/release-controller/_releases/release-okd-scos-4.y-stable.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name":"4-scos-stable",
+  "as": "Stable",
+  "message": "All amd64 OKD/SCOS versions are tracked here",
+  "overrideCLIImage": "image-registry.openshift-image-registry.svc:5000/ocp/4.6:cli",
```

Comment:
I know this is used in all the configs, but it seems wrong to me

Reply:
This is a release-controller specific setting and should remain.  It simply means that the release-controller will use this version of the `cli` image to run it's various `oc adm release ...` commands.

Reply:
I wonder though why we are using an EOL'd release for that? Even 4.6 EUS support ends this month. I'm not saying this has to be changed here now, but it'd be good to have the reasoning behind this documented somewhere.

Reply:
If it aint broke, dont fix it! ;)
In all honesty, the `oc adm release` functionality, we rely on, hasn't changed.  I'll test out bumping up to something more modern and update accordingly.

Reply:
some changes happened for non-single-arch builds that may eventually impact the jobs using an old cli